docker-entrypoint.sh

```diff
@@ -90,6 +90,12 @@ if test -e /var/www/localhost/htdocs/protected/config/dynamic.php &&
 	if [ $SET_PJAX != "false" ]; then
 		sed -i -e "s/'enablePjax' => false/'enablePjax' => true/g" /var/www/localhost/htdocs/protected/config/common.php
 	fi
+
+	if [ "$HUMHUB_TRUSTED_HOSTS" != "false" ]; then
+		sed -i \
+			-e "s|'trustedHosts' => \['.*'\]|'trustedHosts' => ['$HUMHUB_TRUSTED_HOSTS']|g" \
+			/var/www/localhost/htdocs/protected/config/web.php
+	fi
```

Comment:
Not sure but don't think `fi` was supposed to be here was it? :sweat_smile:

Reply:
Although I could be completely wrong and if so my bad :+1:

Reply:
> Not sure but don't think `fi` was supposed to be here was it? 😅

well, the `fi` is definitely supposed to be there since we have a `if` before, but the condition might need to be changed or maybe even removed (ie. set default value to `127.0.0.1` ?).

@mriedmann what do you think ?

Reply:
If you remove `/12` then that would unmask the IP so wouldn't recommend doing that, within the HumHub docs however `/24` is used for reverse proxy :thinking:

https://docs.humhub.org/docs/admin/reverse-proxy/#trusted-hosts